tutorials/configure-vm-hyperv/index.mdx

```diff
@@ -7,21 +7,13 @@ excerpt: |
   Microsoft Hyper-V is a hypervizor which allows to create virtual machines on x86-64 systems. It is available since the release of Windows Server 2012.
```

Comment:
```suggestion
  Microsoft Hyper-V is a hypervizor which allows you to create virtual machines on x86-64 systems. It is available since the release of Windows Server 2012.
```